.dockerignore

```diff
@@ -0,0 +1,6 @@
+#Dockerfile
+Vagrantfile
+.git
+.gitignore
+.env
+.env.template
```

Comment:
This file isn't super important since you are explicitly copying over the relevant folder. That said one more thing that it's recommended to ignore is `**/__pycache__`  - to avoid copying over all of the generated pyc files in todo_app

Reply:
Done